crates/vector-store/src/httproutes.rs

```diff
@@ -489,7 +489,10 @@ fn to_json(value: CqlValue) -> Value {
             TryInto::<Time>::try_into(value)
                 .expect("CqlValue::Time should be correct")
                 .format(&Iso8601::TIME)
-                .expect("Time should be correct"),
+                .expect("Time should be correct")
+                .strip_prefix("T")
+                .unwrap() // Safe to unwrap as format always adds 'T' prefix
```

Comment:
Better to use expect.

```suggestion
                .expect("format should always add 'T' prefix")
```

By the way - there is a problem with this function, because problems with conversions could crash the service. I'm creating a new issue for this.